python/pyspark/ml/tests.py

```diff
@@ -320,6 +320,13 @@ def test_hasparam(self):
         testParams = TestParams()
         self.assertTrue(all([testParams.hasParam(p.name) for p in testParams.params]))
         self.assertFalse(testParams.hasParam("notAParameter"))
+        self.assertTrue(testParams.hasParam(u"maxIter"))
+
+    def test_resolveparam(self):
+        testParams = TestParams()
+        self.assertEqual(testParams._resolveParam(testParams.maxIter), testParams.maxIter)
+        self.assertEqual(testParams._resolveParam("maxIter"), testParams.maxIter)
+        self.assertEqual(testParams._resolveParam(u"maxIter"), testParams.maxIter)
```

Comment:
Would it make sense to also add a param with a unicode name that if it was converted down to ASCII behind the scenes and test it?